src/pages/home/Home.styles.ts

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+export const StudentsContainer = styled.div`
+  width: 100%;
+  padding-top: 20px;
+  min-height: 100vh;
+`;
+export const FiltersContainer = styled.div`
+  display: flex;
+`;
+interface StatusProps {
+  $isActive: boolean;
+}
+export const ButtonContainer = styled.div``;
+export const StatusText = styled.strong<StatusProps>`
+  background-color: ${(props) => (props.$isActive ? "#20C92B" : "#7C7C7C")};
+  color: white;
+  display: inline-block;
+  width: 100px;
+  text-align: center;
+  padding: 2px 15px;
+  font-size: 20px;
+  border-radius: 10px;
+  font-weight: 700;
+`;
```

Comment:
Acá cambié el `<strong> `a una etiqueta tipo `<p>` para que se centre en la columna
```suggestion
export const StatusText = styled.p<StatusProps>`
  background-color: ${(props) => (props.$isActive ? "#20C92B" : "#7C7C7C")};
  color: white;
 display: flex;                  
  align-items: center;          
  justify-content: center;        
  width: 136px;
  height: 32px;
  margin: 0 auto;                
  font-size: 16px;
  border-radius: 10px;
  font-weight: 700;
`;
```